docs/docs/user/what-is-kpops.md

```diff
@@ -1,7 +1,26 @@
 # What is KPOps?
 
-KPOps is a CLI tool that allows deploying and destroying consecutive Kafka Streams applications, Producers, and Kafka 
-Connectors using an easy-to-read and write pipeline definition. KPOps comes with various abstractions that can simplify
-configuring multiple pipelines and steps within pipelines by sharing configuration between different applications,
-like producers or streaming applications. In addition, Kafka apps and its resources can be reset or cleaned up 
-again with a single command.
+With a couple easy commands in the shell and a [`pipeline.yaml`](#example) of under 30 lines, [KPOps](/) can not only [`deploy`](/user/references/cli-commands/#deploy) a pipeline[^1] to a Kubernetes cluster, but also [`reset`](/user/references/cli-commands/#reset), [`clean`](/user/references/cli-commands/#clean) or [`destroy`](/user/references/cli-commands/#destroy) it!
+[^1]:
+     A pipeline can consist of just consecutive [Kafka Streams applications](/user/references/components/#streamsapp) or it could also contain [Producers](/user/references/components/#producer) and/or [Kafka Connectors](/user/references/defaults/#kafkaconnector).
```

Comment:
```suggestion
     A Kafka pipeline can consist of just consecutive [streaming applications](/user/references/components/#streamsapp), [producers](/user/references/components/#producer) and [connectors](/user/references/defaults/#kafkaconnector).
```

Reply:
I would prefer this wording, what do you think?

Reply:
I agree.